vcxagency-artillery/src/messaging/client2agency/msgs-agent.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+const MSGTYPE_GET_MSGS_BY_CONNS = 'did:sov:123456789abcdefghi1234;spec/pairwise/1.0/GET_MSGS_BY_CONNS'
+const MSGTYPE_MSGS_BY_CONNS = 'did:sov:123456789abcdefghi1234;spec/pairwise/1.0/MSGS_BY_CONNS'
+const MSGTYPE_UPDATE_MSG_STATUS_BY_CONNS = 'did:sov:123456789abcdefghi1234;spec/pairwise/1.0/UPDATE_MSG_STATUS_BY_CONNS'
+const MSGTYPE_MSG_STATUS_UPDATED_BY_CONNS = 'did:sov:123456789abcdefghi1234;spec/pairwise/1.0/MSG_STATUS_UPDATED_BY_CONNS'
+
+function buildMsgVcxV2MsgGetMsgsByConns (pairwiseDIDs, uids, statusCodes) {
```

Comment:
Could you also try to rather use dependency on `vcxagency-client` instead of duplicating?

Reply:
I will try. Thanks.